requirements.txt

```diff
@@ -1,9 +1,216 @@
-django-appconf==1.0.1
-django-ses-backend==0.1.1
-Django==1.10
-django_compressor==2.1
-djmail==0.11.0
-futures==3.0.3
-premailer==2.9.6
-psycopg2==2.6.1
-requests==2.9.1
+alabaster==0.7.10
+anaconda-client==1.6.14
```

Comment:
You added some extra packages here and some seem not be required by the application.

Reply:
Use installed anaconda package manager which installs dependencies it assumes we shall need i this project, lets keep to pip for this project

Reply:
It appears that these packages are getting installed automatically or they are dependencies for a another package. I am removing them